staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -665,3 +665,95 @@ function kube::codegen::gen_client() {
             "${input_pkgs[@]}"
     fi
 }
+
+# Generate register code
+#
+# USAGE: kube::codegen::gen_registers [FLAGS] <input-dir>
```

Comment:
Can we make this singular?  `gen_register` ?

Reply:
Just follow the naming convention in this script, for example:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/code-generator/kube_codegen.sh#L65
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/code-generator/kube_codegen.sh#L252

Reply:
There are several helpers, so plural makes sense.

There is one openapi, so singular makes sense.

Are there multiple registers generated?  I guess there could be?

Reply:
Got it, will make it singular.